### PR TITLE
Added in first pass at variables

### PR DESCRIPTION
**Pull request by drfarrell**
Consolidated variables from Figma to make naming consistent and start to establish our design system for future features.

### What is the purpose of this pull request? 

- [X] New feature
- [ ] Documentation update
- [ ] Bug fix
- [ ] Refactor
- [ ] Other

### CODE REVIEW

File: app/tailwind.config.js [hunk 1]
```diff
@@ -51,19 +50,196 @@ module.exports = {
...
+                }],
             },
-            borderRadius: {
+
+            /*--- NUMERIC VALUES –--*/
+            borderRadius: { //to be consolidated
                 lg: 'var(--radius)',
                 md: 'calc(var(--radius) - 2px)',
                 sm: 'calc(var(--radius) - 4px)',
+                'x-small': '2px',
```
Comment (drfarrell) [inline app/tailwind.config.js:236]: Actually I think I'll probably remove these radii – seems a bit over-defined for now.

File: app/tailwind.config.js [hunk 2]
```diff
@@ -51,19 +50,196 @@ module.exports = {
...
                 },
-                surface: {
-                    DEFAULT: '#1a1a1a',
+
+                /*--- TOKENS –--*/
+                bg: {
+                    active: 'var(--bg-tertiary)',
                     brand: {
-                        DEFAULT: 'var(--surface-surface-brand)',
-                        secondary: 'var(--surface-surface-brand-secondary)',
```
Comment (drfarrell) [inline app/tailwind.config.js:58]: Overdefined the variable for bg – removed all of the surfaces and made them bg (also consistent with figma now)

File: app/tailwind.config.js (hunk 1 above)
Comment (Kitenite) [reply app/tailwind.config.js:236]: Less is small afterall unless you think it's going to be used a lot
`xs` is the convention for extra small

File: app/tailwind.config.js [hunk 3]
```diff
@@ -51,19 +50,196 @@ module.exports = {
...
+                        DEFAULT: 'var(--color-red)',
```
Comment (Kitenite) [inline app/tailwind.config.js:58]: We're moving away from red yes?

File: app/tailwind.config.js [hunk 4]
```diff
@@ -51,19 +50,196 @@ module.exports = {
...
+                        secondary: 'var(--color-red-20)',
                     },
+                    default: 'var(--color-gray-100)',
```
Comment (Kitenite) [inline app/tailwind.config.js:61]: I would capitalize default for convention

**Comment by Kitenite** [comment]
Look ok to me. Be sure to test these out or lmk to test a few before marking this ready. I'll do a final pass once you're ready

**Comment by drfarrell** [comment]
This is to address this issue https://github.com/onlook-dev/studio/issues/68

File: app/tailwind.config.js (hunk 3 above)
Comment (drfarrell) [reply app/tailwind.config.js]: I swapped to teal for now – we'll see how it goes. Would like to find a better brand color but we're rollin' with it

File: app/tailwind.config.js (hunk 1 above)
Comment (drfarrell) [reply app/tailwind.config.js]: I just removed it – Tailwind's border radii are pretty solid

**Comment by Kitenite** [comment]
I updated the variables a little. You can see how they're used from the latest changes @drfarrell 

One nitpicky thing (nit): Tailwind variables generally go from lightest 100 to darkest 1000. What do you think about following this pattern with our design? We're currently doing the opposite. 

I would also encourage running the electron app locally when you add variables to make sure it looks alright.

Screenshot 2024-07-17 at 4 21 48 PM

**Review by Kitenite — CHANGES_REQUESTED** [review]
Added some comments. I think it's ready once the comments are addressed

**Comment by drfarrell** [comment]
> I updated the variables a little. You can see how they're used from the latest changes @drfarrell
> 
> One nitpicky thing (nit): Tailwind variables generally go from lightest 100 to darkest 1000. What do you think about following this pattern with our design? We're currently doing the opposite.
> 
> I would also encourage running the electron app locally when you add variables to make sure it looks alright.
> 
> Screenshot 2024-07-17 at 4 21 48 PM

Sure I'm open to this. The original color flow put 700 as one of the brightest options to match Vercel's approach to high-contrast backgrounds, but I think it'd be too complex to expect people to understand that, so I'll adjust the color spread to be more logical.

**Review by Kitenite — APPROVED** [review]
It's working well now. Good to go!

**Pull request merged**